src/humanize/time.py

```diff
@@ -312,14 +322,21 @@ def _quotient_and_remainder(value, divisor, unit, minimum_unit, suppress):
 
     """
     if unit == minimum_unit:
-        return (value / divisor, 0)
+        return value / divisor, 0
     elif unit in suppress:
-        return (0, value)
+        return 0, value
     else:
         return divmod(value, divisor)
 
 
-def _carry(value1, value2, ratio, unit, min_unit, suppress):
+def _carry(
+    value1: int | float,
+    value2: int | float,
+    ratio: int | float,
```

Comment:
It's okay to just have `float` here, Python is fine with passing an `int` to a float argument, and we're likely doing `float` operations on them anyway.

Reply:
Thanks, will replace all the `int | float` with `float.